### PR TITLE
[SPARK-20023] [SQL] Output table comment for DESC FORMATTED

### PR DESCRIPTION
**Pull request by gatorsmile**
### What changes were proposed in this pull request?
Currently, `DESC FORMATTED` did not output the table comment, unlike what `DESC EXTENDED` does. This PR is to fix it.

Also correct the following displayed names in `DESC FORMATTED`, for being consistent with `DESC EXTENDED` 
- `"Create Time:"` -> `"Created:"`
- `"Last Access Time:"` -> `"Last Access:"`

### How was this patch tested?
Added test cases in `describe.sql`

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @cloud-fan @wzhfy

File: sql/core/src/test/resources/sql-tests/inputs/describe.sql [hunk 1]
```diff
@@ -8,6 +8,10 @@ DESC t;
 
 DESC TABLE t;
 
+DESC FORMATTED t;
+
+DESC EXTENDED t;
+
 -- Ignore these because there exist timestamp results, e.g., `Create Table`.
```
Comment (cloud-fan) [inline sql/core/src/test/resources/sql-tests/inputs/describe.sql]: remove this line?

Comment (gatorsmile) [inline sql/core/src/test/resources/sql-tests/inputs/describe.sql]: uh... I did not see that

Comment (gatorsmile) [inline sql/core/src/test/resources/sql-tests/inputs/describe.sql]: Let me enable all of them

File: sql/core/src/test/resources/sql-tests/results/describe.sql.out [hunk 2]
```diff
@@ -79,30 +92,114 @@ d                   	string
...
 -- !query 6 schema
-struct<>
+struct<col_name:string,data_type:string,comment:string>
 -- !query 6 output
+# Detailed Table Information	CatalogTable(
+	Table: `default`.`t`
+	Created: 
+	Last Access: 
+	Type: MANAGED
+	Schema: [StructField(a,StringType,true), StructField(b,IntegerType,true), StructField(c,StringType,true), StructField(d,StringType,true)]
```
Comment (cloud-fan) [inline sql/core/src/test/resources/sql-tests/results/describe.sql.out]: we should improve this, `StructField` is not friendly to users

Comment (cloud-fan) [inline sql/core/src/test/resources/sql-tests/results/describe.sql.out]: we can do it in follow-up

**Comment by cloud-fan** [comment]
LGTM

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**